shell/browser/extensions/electron_extension_loader.cc

```diff
@@ -155,9 +156,10 @@ void ElectronExtensionLoader::LoadExtensionForReload(
     LoadErrorBehavior load_error_behavior) {
   CHECK(!path.empty());
```

Comment:
This whole function should probably be `NOTREACHED()` since we never end up invoking it.

Reply:
I looked more into this and it does end up getting called if an extension invokes `chrome.runtime.reload()`. A callchain starts from `ElectronRuntimeAPIDelegate::ReloadExtension`, to `ExtensionRegistrar::ReloadExtension`, and finally calls `LoadExtensionForReload`.